javascript-bank/bank.js

```diff
@@ -0,0 +1,29 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (isNaN(balance) || balance <= 0 || !Number.isInteger(balance)) return null;
+  var newAcc = new Account(this.nextAccountNumber, holder);
+  newAcc.deposit(balance);
+  this.accounts.push(newAcc);
+  this.nextAccountNumber++;
+  return newAcc.number;
+}
+
+Bank.prototype.getAccount = function (number) {
+  if (number > this.accounts.length || isNaN(number) || !Number.isInteger(number)) return null;
+  return this.accounts[number - 1];
+}
```

Comment:
What happens if `number` is a positive integer, but still doesn't match an existing account?

Reply:
Should return 'null' for anything not an account number.